dv/uvm/core_ibex/riscv_dv_extension/ibex_log_to_trace_csv.py

```diff
@@ -33,7 +33,8 @@
     re.compile(r"^\s*(?P<time>\d+)\s+(?P<cycle>\d+)\s+(?P<pc>[0-9a-f]+)\s+"
                r"(?P<bin>[0-9a-f]+)\s+(?P<instr>\S+\s+\S+)\s*")
 RD_RE = re.compile(r"(x(?P<rd>[1-9]\d*)=0x(?P<rd_val>[0-9a-f]+))")
-ADDR_RE = re.compile(r"(?P<imm>[\-0-9]+?)\((?P<rs1>.*)\)")
+ADDR_RE = re.compile(
+    r"(?P<rd>[a-z0-9]+?),(?P<imm>[\-0-9]+?)\((?P<rs1>[a-z0-9]+)\)")
```

Comment:
Nit: For this sort of thing, I'd probably use Python's implicit string concatenation to split the string literal, rather than do the ugly "hanging indent" thing. As a bonus, you also get to split the regex somewhere meaningful, which makes it a bit easier to read. I'd actually consider splitting it over 3 lines like this:
```python
ADDR_RE = re.compile(r"(?P<rd>[a-z0-9]+?),"
                     r"(?P<imm>[\-0-9]+?)"
                     r"\((?P<rs1>[a-z0-9]+)\)")
```

Reply:
thanks for the suggestion, I'll give that a shot! that definitely does make this regex look a lot cleaner.